### ISSUE TITLE
Import issue with Axios

### ISSUE THREAD

**Issue by rgieseke** [ngyewch/svelte-leaflet#20]
Seems my PR to update Axios (#18, released in 0.9.1) might cause issues in a SvelteKit setup.

I'm getting 
`SyntaxError: The requested module '/node_modules/form-data/lib/browser.js?v=8c61fe2d' does not provide an export named 'default' (at FormData.js?v=8c61fe2d:1:8)` in Chrome

It worked without problems when I previously installed from my branch from GitHub, so I'm not sure what changed.

In Firefox I'm getting `
SyntaxError: ambiguous indirect export: default`

An option might be to replace `axios` with `fetch`:

```svelte
$: {
        if (!geojson) {
            geojson = L.geoJSON(null, options).addTo(getMap());
            eventBridge = new EventBridge(geojson, dispatch, events);
        }
        fetch(url)
            .then((response) => response.json())
            .then(result => {
                geojson.clearLayers();
                geojson.addData(result);
            });
    }
 ```
See https://github.com/rgieseke/svelte-leaflet/tree/remove-axios

I'm not sure if there is a compatibility need to use Axios but maybe this is an option to remove another dependency?
https://caniuse.com/fetch

**Comment by rgfx** [comment]
This messed up my build I can confirm, I getting the same issue. 

![image](https://user-images.githubusercontent.com/2388078/198847373-ac2e37ab-7e02-4ec3-a7ca-60345fddc896.png)

**Comment by rgieseke** [comment]
Another option to solve this might be to leave the fetching out of `svelte-leaflet` and have an object in GeoJSON format as input to `svelte-select`'s GeoJSON component.

**Comment by rgieseke** [comment]
Fixed in #23
